example-setup/README.md

```diff
@@ -14,11 +14,16 @@
 
 These servers are provisioned only with cloud-init/ shell scripts. For service discovery we use consul.
 
-Any node installs consul via cloud-init/ a simple shell script.
-Any node joins a cluster with the first three nodes  
- in the internal network range. 
- 
-Any node has service checks, that announce it's services to the whole cluster.
+- Any node installs consul via cloud-init/ a simple shell script.  
+- Any node joins a cluster with the first three nodes in the internal network range.
+- Consul ACLs are protected by a master token which can is randomly generated.
+The master token can be found in the stack metadata:
+
+```
+openstack stack show <stack_name> | grep output_value
```

Comment:
a cleaner way would be "openstack stack show testACLs-2017-12-14 -f value -c  outputs":
```
syseleven@kickstart:~/dschwabetest/example-setup$ openstack stack show testACLs-2017-12-14 -f value -c  outputs
[
  {
    "output_value": "UbIps0BOIO7jlhoscic40qStSpRn8t", 
    "output_key": "consul_master_token", 
    "description": "No description given"
  }
]
```